apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Tests/CloudTrace/RateLimiterTest.cs

```diff
@@ -69,6 +71,10 @@ public void CanTrace_Multiple()
                 int canTraceCounter = 0;
                 int threads = 10;
 
+                // Start a timer to get the total time from tasks starting to ending.
+                Stopwatch timer = new Stopwatch();
+                timer.Start();
```

Comment:
This is a bit cleaner: `var timer = Stopwatch.StartNew();` (or with the type name if you prefer).